jboss-kie-workbench/added/launch/jboss-kie-workbench.sh

```diff
@@ -157,8 +157,19 @@ function configure_guvnor_settings() {
     if [ "${buildEnableIncremental}" = "true" ] || [ "${buildEnableIncremental}" = "false" ]; then
         JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dbuild.enable-incremental=${buildEnableIncremental}"
     fi
-    # see scripts/jboss-kie-wildfly-common/configure.sh
-    JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.jbpm.designer.perspective=full -Ddesignerdataobjects=false"
+    if [[ $JBOSS_PRODUCT =~ rhpam\-businesscentral(\-monitoring)? ]]; then
+        local jbpmKeystore="${JBOSS_HOME}/standalone/configuration/jBPMKeystore.jceks"
+        if [ -f "${jbpmKeystore}" ]; then
+            # KIECLOUD-218, JBPM-8400
+            JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dkie.keystore.keyStoreURL=file://${jbpmKeystore}"
+            JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dkie.keystore.keyStorePwd=jBPMKeyStorePassword"
```

Comment:
Is expected these values be hardcoded?

Reply:
It is here: https://github.com/kiegroup/kie-wb-distributions/pull/911/files#diff-69e71f5de9b5efa55b63d55aecfe13c9R52
The main point of that PR was just to silence the warnings...

However, I'm trying to better understand the purpose of these properties. You can search for them here: https://docs.jboss.org/jbpm/release/latest/jbpm-docs/html_single/
Perhaps we can just generate the keystore using the controller username and password that we already know (set via template params). I am labeling this PR as "wip" until I figure it out.

Reply:
@spolti The hardcoding to access the truststore and key is not something that I could see a workaround for, however the encrypted value of the kie server and kie controller passwords stored inside are now dynamically set to the correct values.

Reply:
ok, thanks for the explanation.